rotkehlchen/validation/validation_schemas.py

```diff
@@ -0,0 +1,80 @@
+from typing import Any, Dict, Union
+
+from marshmallow import Schema, fields, post_load
+
+from rotkehlchen.api.v1.schemas import UnderlyingTokenInfoSchema
+from rotkehlchen.assets.asset import EthereumToken, UnderlyingToken
+from rotkehlchen.assets.typing import AssetType
+
+
+class AssetListSchema(Schema):
+    identifier = fields.String(required=True)
+    name = fields.String(required=True)
+    symbol = fields.String(required=True)
+    asset_type = fields.String(required=True)
+    forked = fields.String(required=True, allow_none=True)
+    swapped_for = fields.String(required=True, allow_none=True)
```

Comment:
These should be AssetField. We already have verification for them. Use them.

Reply:
This could be missing and fail to validate

Reply:
So `AssetField(load_default=None)`